src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -235,14 +241,14 @@ private BasePanel addNewDatabase(ParserResult result, final Path file, boolean r
 
         if (result.hasWarnings()) {
             JabRefExecutorService.INSTANCE
-                    .execute(() -> ParserResultWarningDialog.showParserResultWarningDialog(result, frame));
+                                          .execute(() -> ParserResultWarningDialog.showParserResultWarningDialog(result, frame));
```

Comment:
why is this on a new thread (which is not the JavaFX one)?

Reply:
No idea, this was probably already present but came to light due to my formatting of the class. But makes indeed no sense